terraform/sqs/sqs.tf

```diff
@@ -7,3 +7,18 @@ resource "aws_sqs_queue" "q" {
 resource "aws_sqs_queue" "dlq" {
   name = "${var.queue_name}_dlq"
 }
+
+resource "aws_cloudwatch_metric_alarm" "dlq_not_empty" {
+  alarm_name = "${aws_sqs_queue.dlq.name}_not_empty"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods = 1
+  metric_name = "NumberOfMessagesSent"
+  namespace = "SQS"
+  period = 60
+  threshold = 1
```

Comment:
Maybe `GreaterThanThreshold` and `threshold = 0`?

I’m not sure it makes a meaningful difference, that’s just what I might have reached for first.